integration/kubernetes/k8s-hugepages.bats

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+issue="https://github.com/kata-containers/runtime/issues/2172"
+
+setup() {
+	skip "test not working see: ${issue}"
+	export KUBECONFIG="$HOME/.kube/config"
+	extract_kata_env
+
+	# Enable hugepages
+	sudo sed -i 's/#enable_hugepages = true/enable_hugepages = true/g' ${RUNTIME_CONFIG_PATH}
+	# Enable sandbox_cgroup_only
+	sudo sed -i 's/sandbox_cgroup_only=false/sandbox_cgroup_only=true/g' ${RUNTIME_CONFIG_PATH}
```

Comment:
Got late to this PR, but...
@GabyCT I think we should add another test that enables hugepages without enabling `sandbox_cgroup_only`, which afaiu currently works ok